code/modules/mob/living/carbon/carbon.dm

```diff
@@ -169,7 +169,7 @@
 				if(start_T && end_T)
 					log_combat(src, throwable_mob, "thrown", addition="grab from tile in [AREACOORD(start_T)] towards tile at [AREACOORD(end_T)]")
 
-	else if((!I.item_flags & ABSTRACT) && !I.has_trait(TRAIT_NODROP))
+	else if(!CHECK_BITFIELD(I.item_flags, ABSTRACT) && !I.has_trait(TRAIT_NODROP))

```

Comment:
why did this break, it should have worked fine before

Reply:
order of operations mistake: `(!I.item_flags & ABSTRACT)` should be `!(I.item_flags & ABSTRACT)`

Reply:
ahh